test/generators/install_generator_test.rb

```diff
@@ -23,10 +23,8 @@ class InstallGeneratorTest < Rails::Generators::TestCase
     run_generator
     assert_file "config/initializers/shopify_app.rb" do |shopify_app|
       assert_match 'config.application_name = "My Shopify App"', shopify_app
-      assert_match "config.api_key = ENV.fetch('SHOPIFY_API_KEY', '').presence " \
-        "|| raise('Missing SHOPIFY_API_KEY')", shopify_app
-      assert_match "config.secret = ENV.fetch('SHOPIFY_API_SECRET', '').presence " \
-        "|| raise('Missing SHOPIFY_API_SECRET')", shopify_app
+      assert_match "config.api_key = ENV.fetch('SHOPIFY_API_KEY', '')", shopify_app
```

Comment:
Rather than copy the exact updated template code into the test, I've simplified this to check for just the 'essential' part. Verifying the template code doesn't prove that the generator works – ideally we should generate an app as part of the test suite.